src/economy/economy.cpp

```diff
@@ -1361,28 +1565,48 @@ void update_single_factory_production(sys::state& state, dcon::factory_id f, dco
 	}
 }
 
-std::tuple<float, float, float> rgo_relevant_population(sys::state& state, dcon::province_id p, dcon::nation_id n) {
+rgo_workers_breakdown rgo_relevant_population(sys::state& state, dcon::province_id p, dcon::nation_id n) {
 	auto relevant_paid_population = 0.f;
 	for(auto wt : state.culture_definitions.rgo_workers) {
 		relevant_paid_population += state.world.province_get_demographics(p, demographics::to_key(state, wt));
 	}
 	auto slaves = state.world.province_get_demographics(p, demographics::to_employment_key(state, state.culture_definitions.slaves));
 
-	return std::tuple<float, float, float>(relevant_paid_population, slaves, relevant_paid_population + slaves);
+	rgo_workers_breakdown result = {
+		.paid_workers = relevant_paid_population,
+		.slaves = slaves,
+		.total = relevant_paid_population + slaves
+	};
+
+	return result;
```

Comment:
it makes no difference, but you can also write `return rgo_workers_breakdown{ ... }` if you want to save a few keystrokes